web/libs/editor/src/stores/AppStore.js

```diff
@@ -880,7 +880,7 @@ export default types
 
         let node;
 
-        while ((node = children.shift())) {
```

Comment:
Not sure why this hasn't caught up to the change you already made to upstream LSF @Travis1282 but maybe since this is such a small change just amend it directly here.